nautobot/cloud/models.py

```diff
@@ -171,7 +171,12 @@ def clean(self):
         # TODO: should we enforce that self.cloud_resource_type.provider == self.cloud_account.provider?
 
 
-@extras_features("graphql")
+@extras_features(
+    "custom_links",
```

Comment:
I *think* we don't need `custom_links` for the through tables since we don't have a UI viewset for them that would display the custom links?